constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -231,6 +245,72 @@ export class ClassEditPage extends React.Component {
                     />
 
+                    <InputText
+                        label="Google Classroom Code"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter the google classroom code"
+                        value={this.state.googleClassCode}
+                        onChangeCallback={(e) =>
+                            this.handleChange(e, "googleClassCode")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
+                    />
+
+                    <InputText
+                        label="Price Lump"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter price for one time payment (Either enter only in this field or only in the price per session field)"
+                        onChangeCallback={(e) =>
+                            this.handleIntegerChange(e, "priceLump")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
+                    />
+
+                    <InputText
+                        label="Price Per Session"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter price for pay per session attended (Either enter only in this field or only in the price lump field)"
+                        onChangeCallback={(e) =>
+                            this.handleIntegerChange(e, "pricePerSession")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
+                    />
+
+                    <InputText
+                        label="Payment Notes"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter payment notes"
+                        value={this.state.paymentNotes}
+                        onChangeCallback={(e) =>
+                            this.handleChange(e, "paymentNotes")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
+                        ]}
```

Comment:
Since payment notes is an optional field and can contain almost anything, I don't think we need any validators. You can just delete this part